nntrainer/src/tensor.cpp

```diff
@@ -61,50 +61,36 @@ static auto rng = [] {
   return rng;
 }();
 
-Tensor::Tensor(const TensorDim d) {
-  dim = d;
-  this->data = std::vector<float>(dim.getDataLen());
-  _is_contiguous = true;
-  setZero();
-}
-
-Tensor::Tensor(int height, int width) {
-  dim.height(height);
-  dim.width(width);
-  this->data = std::vector<float>(dim.getDataLen());
-  _is_contiguous = true;
-  setZero();
+Tensor &Tensor::operator=(Tensor &rhs) {
+  Tensor tmp(rhs);
```

Comment:
is this extra?

Reply:
Implemented incorrectly. Now it is fixed. Thanks.